docs/low-code-spark/gems/subgraph.md

```diff
@@ -8,3 +8,8 @@ todo: true
 ---
 
 Subgraphs allow you to take multiple different Gems and wrap them under a single reusable parent Gem. It allows the user to decompose complex logic into reusable components and simplify the Data Engineering process.
+
+### Subgraph Configurations
+
+Like [Pipeline Configurations](../configuration.md), In Subgraph Configs, User can define values which can be set at Subgraph level and then be accessed inside any component in the Subgraph.
+These will also reflect under Configurations of Pipelines using these Subgraphs, but they can only be edited from Subgraph configs.
```

Comment:
Is there a direct linking from pipeline configs to subgraph configs?
Can job configs overwrite these?